opts.go

```diff
@@ -21,11 +21,24 @@ import (
 	"strings"
 
 	cnilibrary "github.com/containernetworking/cni/libcni"
+	cnitypes "github.com/containernetworking/cni/pkg/types/current"
 	"github.com/pkg/errors"
 )
 
 type CNIOpt func(c *libcni) error
 
+func checkVersion(conf *cnilibrary.NetworkConfig) bool {
+	if (conf.Network.CNIVersion == "") {
```

Comment:
Don't like the idea of masking the error here.

Reply:
why is it ok to be loopback if the version? but if not specified all other network types must match?
@lifubang